examples/ota-provider-app/linux/BUILD.gn

```diff
@@ -29,6 +29,8 @@ executable("chip-ota-provider-app") {
   cflags = [ "-Wconversion" ]
 
   output_dir = root_out_dir
+
+  configs = [ "${chip_root}/examples/ota-provider-app:config" ]
```

Comment:
This should to go on ota-provider-common not the executable

(I really don't understand how this even compiles for you..

oh, actually, I get it.. you didn't change the #includes in BdxOtaSender.cpp and the other files. Please do that)

Reply:
Updated with more fixes after some chat back and forth.